server/app.js

```diff
@@ -0,0 +1,47 @@
+/*
+Server Root File | HOLDING Web App
+GROUP 3: Douglas MacKrell, Briahana Mauge, Joseph P. Pasaoa, Kathy Puma
+*/
+
+
+/* HELPERS */
+const log = console.log;
+
+
+/* MODULE INITS */
+const express = require('express');
+    const app = express();
+    const port = 11000;
+const cors = require('cors');
+    app.use(cors());
+
```

Comment:
Maybe we should use parser here once rather than keep repeating them on likes.js, users.js, photo.js, etc. 
app.use(express.json())
app.use(express.urlencoded({
    extended: false
}))

Reply:
alright, i remember it not working for me in the past without parser on every route but looking back i think i was missing something and might get it to work now. give me half an hour. thanks kathy

Reply:
alright, i remember it not working for me in the past without parser on every route but looking back i think i was missing something and might get it to work now. give me half an hour. thanks kathy

Reply:
got this to work and yas, my ocd side loves the much less code. always learning ;)

done: body parser invocation down to single instance app-wide

Reply:
@KathyPuma please re-check. only changes should be payload to done, body parser on only app.js, and a change from single to double quotes on route templates.